batch7crud-roll4/src/main/java/com/lftechnology/batch7crud/controller/CustomHttpServlet.java

```diff
@@ -51,8 +54,52 @@ public int pageNumber(HttpServletRequest request) {
   }
 
   public int parameterValueAsInt(HttpServletRequest request, int index) {
-    String[] paths = parameterValues(request);
-    return Integer.parseInt(paths[index]);
+    String[] parameters = parameterValues(request);
+    return Integer.parseInt(parameters[index]);
   }
 
+  public void validateUrl(HttpServletRequest request) throws ServletException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "validateUrl" is 12 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)